x-pack/plugins/apm/common/agent_configuration/setting_definitions/general_settings.ts

```diff
@@ -144,7 +139,7 @@ export const generalSettings: RawSettingDefinition[] = [
   {
     key: 'span_frames_min_duration',
     type: 'duration',
-    validation: getDurationRt({ min: -1 }),
+    min: '-1ms',
```

Comment:
Is "-1ms" a valid value - or was -1 allowed to indicate there was no limit?

Reply:
It's a valid value. It's described here:
<img width="531" alt="Screenshot 2020-04-29 at 14 46 00" src="https://user-images.githubusercontent.com/55978943/80597500-3920d780-8a28-11ea-9c97-bc3990315888.png">

Reply:
Great, thanks 👍